src/_P118_Itho.ino

```diff
@@ -87,7 +88,7 @@ volatile bool PLUGIN_118_Int = false;
 
 #define PLUGIN_118
 #define PLUGIN_ID_118         118
-#define PLUGIN_NAME_118       "Itho ventilation remote"
+#define PLUGIN_NAME_118       "Remote - Itho ventilation"
```

Comment:
Doesn't really feel like the same style as other plugin names have.
Looks like there is a separate class of plugins called "remote", but I'm not sure if that really covers this. Does it?

Reply:
Perhaps Communication - Itho ventilation is better suited?

Reply:
Yep, I think so.

Reply:
Implemented